### PR TITLE
Adds known

### PR DESCRIPTION
**Pull request by pierreozoux**

### CODE REVIEW

File: known/content.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#Known: social publishing for groups and individuals
```
Comment (tianon) [inline known/content.md]: This ought to have a space between `#` and `Known`

Comment (tianon) [inline known/content.md]: (Followed by a force push, of course.)

**Comment by pierreozoux** [comment]
@tianon good catch, just fixed ;)

**Comment by pierreozoux** [comment]
Just did a rebase, and all green :)

**Pull request merged**